spec/definitions/ExportSchedule.yaml

```diff
@@ -0,0 +1,103 @@
+type: object
+required:
+  - format
+  - name
+  - recurringInstruction
+  - scheduledStartDate
+properties:
+  id:
+    description: The export identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  status:
+    description: Status of export request
+    readOnly: true
+    type: string
+    enum:
+      - "pending"
+      - "processing"
+      - "completed"
+  name:
+    description: Export name
+    type: string
+  resource:
+    description: The type of resource being exported (eg., transaction).
+    type: string
+    readOnly: true
+  format:
+    description: Export result format
+    type: string
+    enum:
+      - "csv"
+  arguments:
+    description: Export request arguments that can be passed to filter and/or sort the result set
+    type: object
+    properties:
+      filter:
+        description: The collection items filter requires a special format. Use "," for multiple allowed values. Use ";" for multiple fields. See the filter guide for more options and examples about this format.
+        type: string
+      sort:
+        type: array
+        description: The collection items sort field and order (prefix with "-" for descending sort).
+        items:
+          type: string
+      q:
+        type: string
+        description: The partial search of the text fields.
+      criteria:
+        $ref: "#/definitions/Condition"
+  dateRange:
+    type: object
+    $ref: "#/definitions/DateRange"
+  emailNotification:
+    description: List of emails to be notified when export is completed
+    type: array
+    items:
+      type: string
+      format: email
+  recurringInstruction:
+    type: string
+    description: RRule RFC 5545 formatted string used for recurring exports
+  scheduledStartDate:
+    type: string
+    format: date-time
+    description: The date and time for when to begin recurring scheduled exports
+  nextOccurrenceDate:
```

Comment:
I think we should be consistent with the naming here. Perhaps we should call this attribute `nextOccurrenceTime`, since we operate with the exact hour when the export should happen.

Reply:
Should we rename `scheduledStartDate` as well then? Maybe `scheduledStartDateTime` or `scheduledStartTime`?

Reply:
I'd say `scheduledStartTime` to be consistent.